extension/fts/test/test_files/basic.test

```diff
@@ -0,0 +1,346 @@
+-DATASET CSV empty
+
+--
+
+-CASE Basic
+
+-STATEMENT LOAD EXTENSION '${KUZU_ROOT_DIRECTORY}/extension/fts/build/libfts.kuzu_extension';
+---- ok
+
+-STATEMENT CREATE NODE TABLE Book (ID SERIAL, abstract STRING, author STRING, title STRING, PRIMARY KEY (ID));
+---- ok
+-STATEMENT CREATE (b:Book {abstract: 'An exploration of quantum mechanics.', author: 'Alice Johnson', title: 'The Quantum World'});
+---- ok
+-STATEMENT CREATE (b:Book {abstract: 'A magic journey through time and space.', author: 'John Smith', title: 'Chronicles of the Universe'});
+---- ok
+-STATEMENT CREATE (b:Book {abstract: 'An introduction to machine learning techniques.', author: 'Emma Brown', title: 'Learning Machines'});
+---- ok
+-STATEMENT CREATE (b:Book {abstract: 'A deep dive into the history of ancient civilizations.', author: 'Johnson Lee', title: 'Echoes of the Past'});
+---- ok
+-STATEMENT CREATE (b:Book {abstract: 'A fantasy tale of dragons and magic.', author: 'Charlotte Harris', title: 'The Dragon\'s Call'});
+---- ok
+
+-STATEMENT CALL CREATE_FTS_INDEX('Book', 'book_index', ['abstract', 'author', 'title'], stemmer := 'porter');
+---- ok
+-STATEMENT CALL QUERY_FTS_INDEX('Book', 'book_index', 'a quantum machine') RETURN _node.title;
+---- 2
+The Quantum World
+Learning Machines
+-STATEMENT CALL QUERY_FTS_INDEX('Book', 'book_index', 'history') RETURN _node.title;
+---- 1
+Echoes of the Past
+-STATEMENT CALL QUERY_FTS_INDEX('Book', 'book_index', 'past magic world') RETURN _node.title;
+---- 4
+The Quantum World
+Echoes of the Past
+Chronicles of the Universe
+The Dragon's Call
+
+# Build multiple FTS on the same table with same properties and query
+-STATEMENT CALL CREATE_FTS_INDEX('Book', 'book_index_duplicate', ['abstract', 'author', 'title'], stemmer := 'porter');
+---- ok
+-STATEMENT CALL QUERY_FTS_INDEX('Book', 'book_index_duplicate', 'the past magic world') RETURN _node.title;
+---- 4
+The Quantum World
+Echoes of the Past
+Chronicles of the Universe
+The Dragon's Call
+
+# Build multiple FTS on the same table with different properties and query
+-STATEMENT CALL CREATE_FTS_INDEX('Book', 'book_index_small', ['abstract'], stemmer := 'porter');
+---- ok
+-STATEMENT CALL QUERY_FTS_INDEX('Book', 'book_index_small', 'a past magic world') RETURN _node.title;
+---- 2
+Chronicles of the Universe
+The Dragon's Call
+
+-STATEMENT CALL QUERY_FTS_INDEX('Book', 'book_index_small', '') RETURN _node.title;
+---- 0
+-STATEMENT CALL QUERY_FTS_INDEX('Book', 'book_index_small', 'a an the') RETURN _node.title;
+---- 0
+-STATEMENT CALL QUERY_FTS_INDEX('Book', 'book_index_small', 'missing') RETURN _node.title;
+---- 0
+-STATEMENT CALL QUERY_FTS_INDEX('Book', 'book_index', 'johnson') RETURN _node.title;
+---- 2
+The Quantum World
+Echoes of the Past
+-STATEMENT CALL QUERY_FTS_INDEX('Book', 'book_index_small', 'johnson') RETURN _node.title;
+---- 0
+
+-STATEMENT ALTER TABLE Book RENAME author TO author_name;
```

Comment:
i think we already have a test case under `fts_small.test` which tests the renaming of a property with fts index.
You can probably delete this one.